src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -12,68 +13,400 @@
+                                           InstructorAttributes instructor, FeedbackQuestionAttributes question) {
+        FeedbackQuestionEditForm qnForm = new FeedbackQuestionEditForm();
+        qnForm.setAction(new Url(Const.ActionURIs.INSTRUCTOR_FEEDBACK_QUESTION_EDIT));
+        qnForm.setCourseId(instructor.courseId);
+        
+        qnForm.setFeedbackSessionName(feedbackSession.feedbackSessionName);
+        qnForm.setQuestion(question);
+        qnForm.setQuestionDetails(question.getQuestionDetails());
+        qnForm.setQuestionNumberSuffix("-" + String.valueOf(question.questionNumber));
+        
+        qnForm.setNumOfQuestionsOnPage(questions.size());
+        qnForm.setQuestionNumberOptions(getQuestionNumberOptions(questions.size()));
+        qnForm.setQuestionText(question.getQuestionDetails().questionText);
+        
+        FeedbackQuestionGeneralSettings generalSettings = new FeedbackQuestionGeneralSettings();
+        qnForm.setGeneralSettings(generalSettings);
+        generalSettings.setGiverParticipantOptions(getParticipantOptions(question, true));
+        generalSettings.setRecipientParticipantOptions(getParticipantOptions(question, false));
+        
+        
+        // maps for setting visibility
+        Map<String, Boolean> isGiverNameVisible = new HashMap<String, Boolean>();
+        for (FeedbackParticipantType giverType : question.showGiverNameTo) {
+            isGiverNameVisible.put(giverType.name(), true);
+        }
+        generalSettings.setGiverNameVisibleFor(isGiverNameVisible);
+        
+        Map<String, Boolean> isRecipientNameVisible = new HashMap<String, Boolean>();
+        for (FeedbackParticipantType recipientType : question.showRecipientNameTo) {
+            isRecipientNameVisible.put(recipientType.name(), true);
+        }
+        generalSettings.setRecipientNameVisibleFor(isRecipientNameVisible);
+        
+        Map<String, Boolean> isResponsesVisible = new HashMap<String, Boolean>();
+        for (FeedbackParticipantType participantType : question.showResponsesTo) {
+            isResponsesVisible.put(participantType.name(), true);
+        }
+        generalSettings.setResponseVisibleFor(isResponsesVisible);
+        
+        generalSettings.setNumberOfEntitiesToGiveFeedbackToChecked(question.numberOfEntitiesToGiveFeedbackTo == Const.MAX_POSSIBLE_RECIPIENTS);
+        generalSettings.setNumOfEntitiesToGiveFeedbackToValue(question.numberOfEntitiesToGiveFeedbackTo == Const.MAX_POSSIBLE_RECIPIENTS ?
+                                                                 1 :
+                                                                 question.numberOfEntitiesToGiveFeedbackTo);
+        qnForm.setQuestionHasResponses(questionHasResponses.get(question.getId()));
+        
+        List<String> visibilityMessages = question.getVisibilityMessage();
+        generalSettings.setVisibilityMessages(visibilityMessages);
```

Comment:
You can probably just use `question.getVisibilityMessage()` directly here.
If the issue is how it's not plural, then it should be the getter's naming fault and that should be corrected instead.